components/common/prev-next/PrevNext.js

```diff
@@ -20,89 +20,115 @@ import * as React from 'react';
 import { Col } from 'react-bootstrap';
 
 
-import LearnToc from '../../../files1.json';
 import { prefix } from '../../../utils/prefix';
+import { useState } from 'react';
```

Comment:
Move this to line 21. We usually keep all the external packages/libraries at the top.